src/Components/MessageBox.re

```diff
@@ -0,0 +1,71 @@
+open Oni_Core;
+
+open Revery.UI;
+open Revery.UI.Components;
+
+type action('msg) = {
+  label: string,
+  msg: 'msg,
+};
+
+module Styles = {
+  open Style;
+
+  let container = (~theme: Theme.t) => [backgroundColor(theme.background)];
+
+  let message = [
+    padding(20),
+    paddingBottom(30) // I don't know why this is needed, but it is
```

Comment:
There's some bugginess with the content size calculation here which causes the content to overflow unless adding some extra padding.

Reply:
Seems like the text wrapping is what's causing the issue here. If the text doesn't wrap then everything measures as expected.

Reply:
Ah, ya, it sounds like from our conversation there may be a bug with the `measure` method of the `TextNode` 🤔